src/main/scala/is/hail/variant/GenomeReference.scala

```diff
@@ -214,6 +218,12 @@ object GenomeReference {
 
   def hasReference(name: String): Boolean = references.contains(name)
 
+  def deleteReference(name: String): Unit = {
+    if (!hasReference(name))
+      fatal(s"Cannot delete reference genome. Reference genome `$name' does not exist.")
```

Comment:
Same here regarding printing the current names.

Reply:
I prefer add / remove (like `rm`) as complementary operation names, rather than add / delete, but I won't insist if you feel strongly